lib/console/teams/team.ex

```diff
@@ -6,15 +6,21 @@ defmodule Console.Teams.Team do
   @foreign_key_type :binary_id
   schema "teams" do
     field :name, :string
-    has_many :memberships, Console.Teams.Membership
+    field :encryption_version, :binary
+    field :private_key, Cloak.EncryptedBinaryField
+    field :public_key, Cloak.EncryptedBinaryField
+    field :address, :binary
+    field :address_b58, :string
+
+    has_many :memberships, Console.Teams.Membership, on_delete: :delete_all
     many_to_many :users, Console.Auth.User, join_through: "memberships"
-    has_many :invitations, Console.Teams.Invitation
+    has_many :invitations, Console.Teams.Invitation, on_delete: :delete_all
 
-    has_many :devices, Console.Devices.Device
-    has_many :gateways, Console.Gateways.Gateway
-    has_many :channels, Console.Channels.Channel
-    has_many :audit_trails, Console.AuditTrails.AuditTrail
-    has_many :notifications, Console.Notifications.Notification
+    has_many :devices, Console.Devices.Device, on_delete: :delete_all
+    has_many :gateways, Console.Gateways.Gateway, on_delete: :delete_all
+    has_many :channels, Console.Channels.Channel, on_delete: :delete_all
+    has_many :audit_trails, Console.AuditTrails.AuditTrail, on_delete: :delete_all
```

Comment:
Do we want to delete these? might be good to keep

Reply:
later, yeah. for now, I just wanted to be able to clean things up locally without triggering foreign key constraint errors